docs/actions-scripts.md

```diff
@@ -33,17 +112,17 @@ We currently package version 16.1, with `datacheck`, `safetab`, and `safecount`
 
 As Stata is a commercial product, a license key is needed to use it. 
 
-#### If you are a member of the opensafely GitHub organisation
+#### If you are a member of the OpenSAFELY GitHub organisation
```

Comment:
inconsistent with https://github.com/opensafely/documentation/pull/656/files#r832064418

Reply:
I didn't intentially change this, will revert